src/lib/tsreader/DvbUtil.cpp

```diff
@@ -77,7 +77,7 @@ namespace MPTV
     //*******************************************************************
     uint32_t crc32(char *data, int len)
     {
-        int i;
+        register int i;
```

Comment:
Was a reason to add regster? Seen since C++11 deprecated and with C++17 no more used.

Reply:
Not sure at all here.

Reply:
Ok, removed.